src/api/config.c

```diff
@@ -612,6 +612,49 @@ m64p_error ConfigShutdown(void)
 /* Selector functions, exported outside of the Core */
 /* ------------------------------------------------ */
 
+EXPORT m64p_error CALL ConfigExternalGetParameter(const char *FileName, const char *SectionName, const char *ParamName, char* ParamPtr, int ParamMaxLength)
+{
+    FILE *fPtr;
+    char buffer[ParamMaxLength + 64]; //ParamMaxLength is the size of the value, we add 64 to make sure we have enough room to store the rest of the line.
```

Comment:
Ok I'll change it to malloc/free

> In this situation I would rather use a fixed length for buffer, being the max size of a line, maybe 4096 like we discussed before?

What if the plugin specifies a value for ParamMaxLength that is larger than 4096?